action.yml

```diff
@@ -176,10 +176,18 @@ inputs:
     description: 'Display some statistics at the end regarding the stale workflow (only when the logs are enabled).'
     default: 'true'
     required: false
+  labels-to-add-when-stale:
+    description: 'A comma delimited list of labels to add when a issue or pull request receives activity and has the stale-issue-label or stale-pr-label add to it.'
+    default: ''
+    required: false
```

Comment:
I agree with @MaksimZhukov, instead of creating this new input we can expand functionality of inputs: stale-pr-label and stale-issue-label to get the array of labels which user wants to apply to the issue/PR in case it becomes stale;